test/test_automl/test_estimators.py

```diff
@@ -668,18 +668,20 @@ def test_conversion_of_list_to_np(self, fit_ensemble, refit, fit):
         self.assertIsInstance(fit.call_args[0][1], np.ndarray)
         automl.refit(X, y)
         self.assertEqual(refit.call_count, 1)
-        self.assertIsInstance(refit.call_args[0][0], np.ndarray)
-        self.assertIsInstance(refit.call_args[0][1], np.ndarray)
         automl.fit_ensemble(y)
         self.assertEqual(fit_ensemble.call_count, 1)
         self.assertIsInstance(fit_ensemble.call_args[0][0], np.ndarray)
 
     def test_classification_pandas_support(self):
         X, y = sklearn.datasets.fetch_openml(
-            data_id=37,  # diabetes
+            data_id=40981,  # cat/num Australian
             return_X_y=True,
             as_frame=True,
         )
+
+        # Drop NAN!!
```

Comment:
Does this drop empty columns or what exactly does it drop?

Reply:
Yes, it drops columns! Because the dataset 2 has a couple of columns that are completely nan. I will debug the problems with dataset 2 and base on the outcome see if it can be incorporated. If we need another change before, I will create another PR